src/empire/empire_xml.c

```diff
@@ -281,11 +319,15 @@ static void XMLCALL xml_start_element(void *unused, const char *name, const char
     if (strcmp(name, "city") == 0) {
         xml_parse_city(num_attrs, attributes);
     } else if (strcmp(name, "sells") == 0) {
-        data.current_city_resource_list = LIST_SELLS;
+        data.current_city_list = LIST_SELLS;
     } else if (strcmp(name, "buys") == 0) {
-        data.current_city_resource_list = LIST_BUYS;
+        data.current_city_list = LIST_BUYS;
+    } else if (strcmp(name, "trade_points") == 0) {
```

Comment:
I think something like `path_waypoints` would be a more meaningful name.

Reply:
I didn’t want to cause confusion with invasion battle paths.

Reply:
Ok, `trade_waypoints` then.